tests/Tests_RangeEnhancedIterator.py

```diff
@@ -0,0 +1,44 @@
+
+# coding: utf-8
+
+# In[ ]:
+
+
+import unittest
+import RangeEnhancedIterator_Generator
+from RangeEnhancedIterator_Generator import IteratorEnhancedRange
+
+
+class IteratorTests(unittest.TestCase):
+
+    def setUp(self):
+        print ('Setup called')
+    
+    def test_on_range_w_one_arg(self):
```

Comment:
Could we use the same set of tests for both iterator and generator?

Reply:
No, because I use the `next` function in the tests for iterator, which I cannot do for the generator tests.